nixos/lib/test-driver/test-driver.py

```diff
@@ -389,7 +390,7 @@ def succeed(self, *commands):
                     raise Exception(
                         "command `{}` failed (exit code {})".format(command, status)
                     )
-                return output
+        return output
```

Comment:
the original perl code does `$res .= $out;` and then in the end `return $res`. We should do this accumulation, too.